test/data/event.get.full.yaml

```diff
@@ -0,0 +1,24 @@
+# Event Get Example
+id: bbf22a3808c19dc50777258a253805b14fb3ad8b
+builds:
+    - 8843d7f92416211de9ebb963ff4ce28125932878
+commit:
+    message: Fixing a bug with signing
+    author:
+        url: https://github.com/stjohnjohnson
+        name: St. John Johnson
+        username: stjohnjohnson
+        avatar: https://avatars.githubusercontent.com/u/622065?v=3
+    url: https://github.com/scredriver-cd/screwdriver/commit/46f1a0bd5592a2f9244ca321b129902a06b53e03
+createTime: "2038-01-19T03:14:08.131Z"
+creator:
+    name: St. John Johnson
```

Comment:
I know this is test but could we make this a different user to make it more obvious that creator != committer.